packages/mold/build.sh

```diff
@@ -0,0 +1,45 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/rui314/mold
+TERMUX_PKG_DESCRIPTION="mold: A Modern Linker"
+TERMUX_PKG_LICENSE="AGPL-V3"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.0.3
+_COMMIT=6ffcae428622c216b5f77384a7456fa248981d7c
```

Comment:
The tag `v1.0.3` seems to be for commit `bda68f9` in `release/1.0.3` branch. Any reason why `6ffcae4` should be specified instead? If any, then I suggest that `TERMUX_PKG_VERSION` include the commit date, such as

```bash
TERMUX_PKG_VERSION=2022.02.05
```

Reply:
https://github.com/rui314/mold/commit/6ffcae428622c216b5f77384a7456fa248981d7c
and
https://github.com/rui314/mold/commit/03e0cef726571fa0f6435ae0ef9d4146a285b1f8
are needed to fix
https://github.com/rui314/mold/pull/311
which unique to us as we use clang on device

I disagree with using date formats as its greater than a potential 1.0.4
I could try tweak to the version highlight this

Reply:
Stay on 1.0.3 and add 6ffcae4 and 03e0cef as patches, specifying 1.0.3 and using a newer commit is a bit misleading

> I disagree with using date formats as its greater than a potential 1.0.4

This is solved using an [epoch](https://www.debian.org/doc/debian-policy/ch-controlfields.html#version), i.e. to switch back to 1.0.4 from a date we use `TERMUX_PKG_VERSION=1:1.0.4`